svgnative/src/SVGStringParser.cpp

```diff
@@ -391,6 +391,7 @@ void ParsePathString(const std::string& pathString, Path& p)
     float prevControlY{}; // nanf(nullptr);
     float prevCurvePointX{};
     float prevCurvePointY{};
+    char lastCommand = 'm';
```

Comment:
`prev` was meant to represent the last command, is there a reason why we need to store the info a 2nd time?

Reply:
`prev` is the previous character here, which is the current parsing command. Parsing "T" / "t" command requires information of the previous command, so we store it in the `lastCommand` variable.

Reply:
Ah ok. That makes sense.

LGTM